roles/manage-aws-infra/tests/sample.aws.example.com.d/inventory/ec2.ini

```diff
@@ -0,0 +1,191 @@
+[ec2]
```

Comment:
Please remove this "extra" inventory as it's pretty much a duplicate of the top level `inventory/sample.aws...` inventory. If you'd like to keep this `tests` directory, I'd suggest linking the top level inventory, but in my opinion this `tests` directory could probably just be removed.